CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+# v5.0.0 (Thu Jan 02 2025)
```

Comment:
As there's a breaking change, I updated the version to v5. Happy to change that though.